genesis/process/shardGenesisBlockCreator.go

```diff
@@ -453,22 +460,32 @@ func deployInitialSmartContract(
 	return deployProc.Deploy(sc)
 }
 
-func incrementStakersNonces(arg ArgsGenesisBlockCreator) (int, error) {
+func incrementStakersNonces(processors *genesisProcessors, arg ArgsGenesisBlockCreator) (int, error) {
```

Comment:
Is it still one nonce increase if for example one account does staking for multiple nodes?

Reply:
good point. will change.

Reply:
done